### DISCUSSION TITLE
connection refused error (mint)

### DISCUSSION THREAD

**Discussion by GameParrot** [rustdesk/rustdesk#6909] [General]
### Bug Description

I cannot connect to the desk with pernament password, it says connection refused

### How to Reproduce

1. Set pernament password
2. Try to login

### Expected Behavior

You should login

### Operating system(s) on local side and remote side

linux mint 21.3 (ubuntu 22.04 base)

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/85067619/c8a101b1-6c1f-43f2-8f0b-4df3397bd372)

**Comment by rustdesk** [comment]
Actually it is x11 connection refuse.

**Comment by rustdesk** [comment]
tracked by https://github.com/rustdesk/rustdesk/issues/3565#:~:text=error%C2%A0%236896-,mint,-Flickering%20On%20Linux
